File: app/src/main/java/nl/whitedove/agecalculator/Persoon.java
package nl.whitedove.agecalculator;

import org.joda.time.DateTime;

class Persoon {

    private int id;
    private String naam;
    private DateTime gebdatum;
    private Boolean geselecteerd;
    private Boolean getoond;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    String getNaam() {
        return naam;
    }

    void setNaam(String naam) {
        this.naam = naam;
    }

    DateTime getGebdatum() {
        return gebdatum;
    }

    void setGebdatum(DateTime gebdatum) {
        this.gebdatum = gebdatum;
    }

    Boolean getGeselecteerd() {
        return geselecteerd;
    }

    void setGeselecteerd(Boolean geselecteerd) {
        this.geselecteerd = geselecteerd;
    }

    Boolean getGetoond() {
        return getoond;
    }

    void setGetoond(Boolean getoond) {
        this.getoond = getoond;
    }
}

File: app/src/main/java/nl/whitedove/agecalculator/Persoon.kt
package nl.whitedove.agecalculator

import org.joda.time.DateTime

internal class Persoon {
    private var id = 0
    private var naam: String? = null
    private var gebdatum: DateTime? = null
    private var geselecteerd: Boolean? = null
    private var gevinkt: Boolean? = null
    private var getoond: Boolean? = null
    fun getId(): Int {
        return id
    }

    fun setId(id: Int) {
        this.id = id
    }

    fun getNaam(): String? {
        return naam
    }

    fun setNaam(naam: String?) {
        this.naam = naam
    }

    fun getGebdatum(): DateTime? {
        return gebdatum
    }

    fun setGebdatum(gebdatum: DateTime?) {
        this.gebdatum = gebdatum
    }

    fun getGeselecteerd(): Boolean? {
        return geselecteerd
    }

    fun setGeselecteerd(geselecteerd: Boolean?) {
        this.geselecteerd = geselecteerd
    }

    fun getGetoond(): Boolean? {
        return getoond
    }

    fun setGetoond(getoond: Boolean?) {
        this.getoond = getoond
    }

    fun getGevinkt(): Boolean? {
        return gevinkt
    }

    fun setGevinkt(gevinkt: Boolean?) {
        this.gevinkt = gevinkt
    }
}